src/store/undoRedo/index.ts

```diff
@@ -29,17 +54,39 @@ export const getters: GetterTree<IUndoRedoState, IRootState> = {
     const isNotLastPosition = state.position < state.snapshots.length - 1
     return hasMoreThanInitialSnapshot && isNotLastPosition
   },
+  currentSnapshot(state): ISnapshot {
+    return state.snapshots[state.position]
+  },
+  previousSnapshot(state): ISnapshot | null {
+    return state.snapshots[state.position - 1] ?? null
+  },
+  futureSnapshot(state): ISnapshot | null {
+    return state.snapshots[state.position + 1] ?? null
+  },
```

Comment:
👍
I guess we can rewrite `canUndo` in the future, like `return this.previousSnapshot !== null`
And the same for `canRedo`